package.json

```diff
@@ -52,10 +45,7 @@
     "@iarna/toml": "2.2.5",
```

Comment:
I think this was used by a script that I wrote for updating TOML files during release, but not sure it ever made it. I think this can go away in any case.